docs/JSON-RPC.md

```diff
@@ -471,3 +526,43 @@ Parameters:
 | params | object | No parameters (empty object). |
 
 
+### jamulusclient/recorderState
+
+Emitted when the client is connected to a server who's recorder state changes.
```

Comment:
It only gets emitted when the client is connect to a server and that server's recorder state changes.  I think the first part is probably redundant -- i.e. it can be assumed that a client won't be informed of state changes to servers to which it isn't connected.

Reply:
Fixed in next commit.